packages/types/lib/logs/messages.ts

```diff
@@ -134,6 +134,7 @@ export interface MessageRow {
     response: {
         code: number;
         headers: Record<string, string>;
+        body?: unknown;
```

Comment:
Just had a thought before going to bed, you need to add this prop to schema.ts as keyword "index": false and "doc_values": false otherwise it will blow up

Reply:
Fixed in 2c9e4f96